Essentials/src/com/earth2me/essentials/Settings.java

```diff
@@ -826,7 +826,11 @@ private boolean _changeDisplayName() {
     public boolean changeDisplayName() {
         return changeDisplayName;
     }
-
+    
+    public boolean storeIPAddressPlainText() {
```

Comment:
Method name implies that it will store the IP on some other form other than plain-text, should be renamed for clarity.